docs/toc.yml

```diff
@@ -692,7 +693,7 @@
     - name: Tutorials
       items:
       - name: Build your first custom functions add-in
-        href: quickstarts/excel-custom-functions-quickstart-redirect.md
```

Comment:
I don't think this is being used anywhere else and so, since this wasn't an actual file, we should be able to delete the entry from the redirect file, right?

Reply:
Makes sense to me.